.github/actions/verify-token/src/index.ts

```diff
@@ -127,29 +133,40 @@ async function run(): Promise<void> {
       rawTokenObj.tool.actions.build_artifacts.path
     );
 
+    // No validation needed for the builder inputs,
+    // they may be empty.
+    // TODO(#1780): test empty inputs.
+
+    // Extract certificate information.
+    const [toolURI, toolRepository, toolRef, toolSha, toolPath] =
+      parseCertificate(bundle);
+
+    // Extract the inputs.
+    // See https://github.com/slsa-framework/slsa-github-generator/issues/1737.
+    const rawFinalTokenObj = await filterWorkflowInputs(
```

Comment:
nit: it's not actually final

```suggestion
    const rawFilteredTokenObj = await filterWorkflowInputs(
```